lib/util/backend.dart

```diff
@@ -94,105 +107,24 @@ class Backend {
       parseObject.set(key, value);
     });
 
-    return BackendResponse(await parseObject.save());
+    try {
+      return jsonDecode((await parseObject.save()).results!.first.toString());
+    } catch (e) {
+      rethrow;
+    }
```

Comment:
These try catch statements are unnecessary since the exception thrown are automatically passed to the caller.